bin/build

```diff
@@ -2,31 +2,43 @@
 
 set -e
```

Comment:
set -euo pipefail

i know it is not your code, and yet

Reply:
updating now- thanks!